src/constants.ts

```diff
@@ -12,7 +12,11 @@
 */
 
 export default {
+  CLIENT_ID: '151109002640-6hsca3c8lv5eutorkavk157rle1rjsgt.apps.googleusercontent.com',
+  DRIVE_FOLDER: 'Snapshot Photos',
   IMAGE_TYPE: 'image/jpeg',
-  SUPPORTS_IMAGE_CAPTURE: 'ImageCapture' in window,
+  SUPPORTS_BGSYNC: 'SyncManager' in self,
+  SUPPORTS_IMAGE_CAPTURE: 'ImageCapture' in self,
```

Comment:
Even though it works, it is only exposed in the `window` Object.

Reply:
`constants.ts` gets included by the service worker now, so `window` isn't necessarily defined. I changed it to `self` to avoid an error in the service worker, but it shouldn't change the validity of the test.